src/network/networkd-link.h

```diff
@@ -125,6 +125,8 @@ typedef struct Link {
         unsigned lldp_tx_fast; /* The LLDP txFast counter (See 802.1ab-2009, section 9.2.5.18) */
         sd_event_source *lldp_emit_event_source;
 
+        sd_event_source *address_event_source;
```

Comment:
This seems not unref()ed. Please add `sd_event_source_unref()` at `link_free()`.